apidoc/Titanium/UI/Toolbar.yml

```diff
@@ -32,20 +55,36 @@ properties:
 
         For information about color values, see the "Colors" section of <Titanium.UI>. 
     type: String
+    platforms: [iphone, ipad]
     
   - name: items
     summary: An array of buttons (or other widgets) contained in the toolbar.
     type: Array<Titanium.UI.View>
+    platforms: [iphone, ipad]
     
   - name: borderTop
-    summary: If `true`, a border is drawn on the top of the toolbar.
+    summary: If `true`, a border is drawn on the top of the toolbar. This property is ignored on iOS 7 and above.
     type: Boolean
     default: true
-    
+    platforms: [iphone, ipad]
+   
   - name: borderBottom
-    summary: If `true`, a border is drawn on the bottom of the toolbar.
+    summary: If `true`, a border is drawn on the bottom of the toolbar. This property is ignored on iOS 7 and above.
```

Comment:
Actually, we should use `#if defined(USE_TI_UIIOSTOOLBAR) || defined(USE_TI_UITOOLBAR)` all around the place, since Ti.UI.iOS.Toolbar will invoke TiUIToolbarProxy internally as well.